news/+uv.internal

```diff
@@ -0,0 +1 @@
+Use `uv` to manage project and project dependencies. @ericof
```

Comment:
uv is not an inline literal, but its actual brand name. `uv` is used in commands.

```suggestion
Use uv to manage project and project dependencies. @ericof
```